.gitlab/deploy_packages/nix.yml

```diff
@@ -219,26 +219,61 @@ deploy_staging_iot_agent:
     - $S3_CP_CMD ./agent $S3_DSD6_URI/linux/iot/agent-$PACKAGE_VERSION --grants read=uri=http://acs.amazonaws.com/groups/global/AllUsers full=id=3a6e02b08553fd157ae3fb918945dd1eaae5a1aa818940381ef07a430cf25732
 
 # Datadog Installer
+
+# Common jobs
+
+# The installer is a special case because it's built by the datadog-agent pipeline, but
+# is expected to be deployed in a dedicated folder, so we need to define a special base which
+# fetches packages from a custom value of S3_RELEASE_ARTIFACTS_URI as the default value
+# include datadog-agent as a product
+.deploy_installer_deb:
+  rules:
+    !reference [.on_deploy]
+  resource_group: deb_bucket
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-builders/gitlab_agent_deploy:$DATADOG_AGENT_BUILDERS
+  stage: deploy_packages
+  tags: ["arch:amd64"]
+  before_script:
+    - ls $OMNIBUS_PACKAGE_DIR
+  script:
+    - $S3_CP_CMD --recursive --exclude "*" --include "*_7.*${PACKAGE_ARCH}.deb" "$OMNIBUS_PACKAGE_DIR" "s3://dd-release-artifacts/datadog-installer/$CI_PIPELINE_ID/deb/${PACKAGE_ARCH}/"
+
+.deploy_installer_rpm:
+  rules:
+    !reference [.on_deploy]
+  resource_group: rpm_bucket
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-builders/gitlab_agent_deploy:$DATADOG_AGENT_BUILDERS
+  stage: deploy_packages
+  tags: ["arch:amd64"]
```

Comment:
shouldn't the arch tag be overwritten below?

Reply:
This is the runner arch, not the package one. We *could* run these jobs on arm (maybe we should actually) but this is orthogonal to the arch of the package which is being deployed